src/main/java/Parser.java

```diff
@@ -0,0 +1,101 @@
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
```

Comment:
I think it could be good practice to add a line spacing between import statements from different packages!